src/hotspot/cpu/x86/globals_x86.hpp

```diff
@@ -231,8 +231,10 @@ define_pd_global(intx, InitArrayShortSize, 8*BytesPerLong);
                                                                             \
   product(bool, IntelJccErratumMitigation, true, DIAGNOSTIC,                \
              "Turn off JVM mitigations related to Intel micro code "        \
-             "mitigations for the Intel JCC erratum")
-
+             "mitigations for the Intel JCC erratum")                       \
+                                                                            \
+  product(bool, UseAPX, false, EXPERIMENTAL,                                              \
```

Comment:
Spacing to `\`

Reply:
@steveatgh, do you plan to add code to `vm_version_x86.*` to check for presence of AVX and setting this flag accordingly?

Reply:
Thanks @vnkozlov.  
Spacing fixed locally.
Regarding the UseAPX flag, yes, a subsequent PR (see JDK-8329030) will tie the logic of the flag in with querying the hardware features.

Reply:
Okay.